packages/office-ui-fabric-react/src/components/ProgressIndicator/ProgressIndicator.scss

```diff
@@ -55,13 +55,44 @@ $ProgressIndicatorTextHeight: 18px;
   transition: width .3s ease;
   width: 0;
 
+
+  &.indeterminate {
+    position: absolute;
+    min-width: 33%;
+    background: linear-gradient(to right, transparent 0%, $ms-color-themePrimary 50%, transparent 100%);
+    animation: indeterminateProgress 3s infinite;
+  }
+
   @include high-contrast {
     background-color: WindowText;
   }
 }
+[dir='rtl'] .progressBar {
```

Comment:
We actually have a mixin in _i18n.scss that we normally use for this scenario. 

`
@include RTL {
 .progressBar {
 ...
 }
}
`
Or something similar